### ISSUE TITLE
Add a feed?

### ISSUE THREAD

**Issue by tfausak** [feramhq/feramhq.github.io#10]
Hi! I am the editor of Haskell Weekly. I would like to be notified when you publish a new post, but as far as I can tell your site does not have a feed. Would you mind adding an Atom or RSS feed? Thanks!

**Comment by ad-si** [comment]
Hi Taylor (I know who you are 😜),
thanks for reaching out. Unfortunately I couldn't help myself and basically implemented a blogging engine with shake and pandoc. This means the build process for the blog is pretty awesome now 😁,
but it's not easy to add an Atom feed as I'll have to implement a lot of it myself 😅.
It's definitely on the Roadmap, however!
Until then, you can sign up for our newsletter instead (at the bottom of https://www.feram.io).
Sorry for the inconvenience!

**Comment by tfausak** [comment]
No problem! I understand completely. Haskell Weekly itself has a bespoke blogging engine. Fortunately rendering Atom or RSS feeds isn't too complicated. Feel free to use my code as an example! https://github.com/haskellweekly/haskellweekly.github.io/blob/1018f6e655c377d2e4a0ccca689994945e457896/executables/haskell-weekly.hs#L259

**Comment by ad-si** [comment]
Wow, cool, you even went one step further and only included cmark as external dependency.
I'll definitely check out the code, thanks!

**Comment by tfausak** [comment]
Yup, I got tired of waiting up to an hour for a bunch of dependencies to build. My needs were simple, so I felt justified building yet another static site generator 😆 https://github.com/haskellweekly/haskellweekly.github.io/pull/84

**Comment by ad-si** [comment]
Haha, I know what you mean. This blog takes 45 mins to build (cold). I haven't yet had the time to figure out how to cache it on Travis CI, so currently I'm building locally and publish by committing the HTML 😂😅. (Or just pushing and waiting the time if it's not crucial …)

**Comment by ad-si** [comment]
Closing this for now, as I don't expect to reactivate the Feram Blog again any time soon.
Anything interesting I'm doing will be blogged about on https://adriansieber.com/atom.xml and https://blog.airsequel.com/feed.xml.
